contracts/DixelClubV2NFT.sol

```diff
@@ -38,13 +38,21 @@ contract DixelClubV2NFT is ERC721Enumerable, Ownable, SVGGenerator {
         uint24[PALETTE_SIZE] palette; // 24bit color (16,777,216) - up to 16 colors
     }
 
+    struct PosAndC {
+        uint96 Position;
+        uint96 Count;
+        bool init;
```

Comment:
I guess we don't need `init` separately because `count == 0` is basically "not in whitelist"

Reply:
What I considered when I wrote that is, This is to make sure that the user is on the AllowList and has no lefted the count.
I didn't implement `init` at first, so it's good to think of it as additional information.